WickedEngine/shaders/emittedparticle_sphforceCS.hlsl

```diff
@@ -115,8 +115,7 @@ void main( uint3 DTid : SV_DispatchThreadID, uint groupIndex : SV_GroupIndex, ui
 
 								f_a += mass * ((pressureA + pressureB) / (2 * densityA * densityB)) * W * rNorm;
 
-								float r3 = r2 * r;
-								W = -(r3 / (2 * h3)) + (r2 / h2) + (h / (2 * r)) - 1; // laplacian smoothing function
```

Comment:
Sorry, I've been very busy. But I don't understand, why is the laplacian smoothing function completely removed now from the equations?

Reply:
I think the problem is that I don't understand what the triangle means in this: 
![image](https://user-images.githubusercontent.com/12954849/148639872-914250ae-a086-4ced-ac3a-0127784905e9.png)

Reply:
The triangle (nabla) squared is the Laplace operator. When you apply it to the viscosity kernel above you end up with the expression I posted in the issue earlier. This shows it more explicitly:

![image](https://user-images.githubusercontent.com/18217298/148995992-c01a3d87-deaa-4ea7-82db-6c6423d7dd60.png)

So the Laplacian of the smoothing function is applied, but I removed the comment. Sorry for the confusion.

Reply:
Thanks. Since it's more closely matches the original article's implementation, this will be merged. I will probably modify the current fluid simulation sample's viscosity to better match the original look. And probably better default values for viscosity should be used too.